src/ddoc/highlight.d

```diff
@@ -133,6 +133,28 @@ $(D_CODE $(D_KEYWORD unittest) {
 	assert(r1 == e1, r1);
 }
 
+unittest {
+	auto s = `
+        ---------
+        asm pure nothrow @nogc @trusted
+        {
+            // the compiler does not check the attributes
+            ret;
+        }
+        ---------
+`;
+	auto e = `
+        $(D_CODE $(D_KEYWORD asm) $(D_KEYWORD pure) $(D_KEYWORD nothrow) @nogc @trusted
```

Comment:
I haven't checked the implementation, but it seems intentional to me.
It makes sense to support code such as [this](https://github.com/D-Programming-Language/dlang.org/blob/4537710e5b8239764f989dc59a10f69419b2bc68/changelog.dd#L212), which is the code the image is extracted from.